tests/integration/weaviate_memory_tests.py

```
@@ -52,7 +52,7 @@ def setUpClass(cls):
     def setUp(self):
         try:
             self.client.schema.delete_class(self.index)
-        except:
+        except Exception:
             pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)